### DISCUSSION TITLE
0% code coverage with Quasar (Vue.js) and Cypress

### DISCUSSION THREAD

**Discussion by maiolica** [cypress-io/cypress#20238] [Questions and Help]
Hello everybody, I've some issues instrumenting Vue.js components in Quasar.

https://github.com/maiolica/quasar-2-test-coverage

In this repo I've set the default Quasar 2 installation with Cypress e2e tests and the the Cypress code coverage plugin, but it seems .vue files are not instrumented correctly, may someone help me?

This issue seems similar to: https://github.com/istanbuljs/nyc/issues/1366

Default setting:
<img width="669" alt="image" src="https://user-images.githubusercontent.com/771265/154472385-2cbfcd98-5a81-4767-9cff-3d5cef8a872e.png">

Tweaked .nycrc settings for including .vue files
<img width="438" alt="image" src="https://user-images.githubusercontent.com/771265/154471924-86987a55-28b3-42bf-9784-556dcafa6ca0.png">

**Comment by IlCallo** [comment]
From what I understood, 0% coverage can happen either when the code hasn't been correctly instrumented or when the code isn't actually used in the run (eg. typescript or webpack dropped it since it wasn't used anywhere)
In those cases, you'll still get a line when using `all` option, but it will be marked in grey and statements will be `0/0` since none of them are actually used

**Comment by IlCallo** [comment]
Just published oob code coverage support for Cypress + Vite + Quasar, check it out: https://github.com/quasarframework/quasar-testing/releases/tag/e2e-cypress-v4.1.0
